classes/PodsAdmin.php

```diff
@@ -174,6 +174,18 @@ public function admin_head() {
 			}//end if
 		}//end if
 
+		/**
+		 * Filter to disable default loading of the DFV script. By default, Pods
+		 * will always enqueue the DFV script if is_admin()
+		 *
+		 * @param bool Whether or not to enqueue by default
+		 *
+		 * @since 2.7.10
+		 */
+		if ( apply_filters( 'pods_default_enqueue_dfv', is_admin() ) ) {
```

Comment:
You can set this as `true` since the action only runs in the admin area already.